bucket/fastfetch.json

```diff
@@ -0,0 +1,24 @@
+{
+    "version": "1.8.2",
+    "description": "Like neofetch, but much faster because written in C",
+    "homepage": "https://github.com/LinusDierheimer/fastfetch",
+    "license": "Freeware",
```

Comment:
Please change to `MIT` identifier (should be a valid [SPDX Identifier](https://spdx.org/licenses) as per Contributing Guide).